openghg/util/_user.py

```diff
@@ -126,9 +126,24 @@ def create_config(silent: bool = False) -> None:
 
         if new_shared_stores:
             existing = [k for k in new_shared_stores if k in stores]
+
+            existing_paths = [path["path"] for path in stores.values()]
```

Comment:
```suggestion
            existing_paths = [path_data["path"] for path_data in stores.values()]
```

Reply:
I think `path_data` makes it clearer what's happening here.

Reply:
done